### DISCUSSION TITLE
REST data source environment variables

### DISCUSSION THREAD

**Discussion by benaunt** [Budibase/budibase#11120] [Ideas]
Hi,

When using MSSQL as a data source you have the option to link to environment variables for all the connection fields (host, db, username password etc).

It would be useful if REST API queries can also point to an environment variable for all or part of the address in a rest query.

This will allow the app preview to point to a Dev/Test api and the live app to point to production API's. 

Thanks

**Comment by melohagan** [comment]
Hey @benaunt 

Good idea! I'll share it with the rest of the team.

**Reply by melohagan** [reply]
@benaunt It appears that this may actually be possible. Can you put a binding in your REST API URL such as `http://api/path/{{ env.MY_ENV_VAR }}` ?

**Reply by benaunt** [reply]
This worked, still might be worth making the functionality apparent via the UI.
